pom.xml

```diff
@@ -1312,6 +1313,11 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
         <artifactId>nimbus-jose-jwt</artifactId>
         <version>${com.nimbusds.nimbus-jose-jwt.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-annotations</artifactId>
+        <version>${spotbugs.annotation.version}</version>
+      </dependency>
```

Comment:
`spotbugs-annotations` has LGPL license, we cannot use it.  Suppress the issue in `hadoop-ozone/ozone-manager/dev-support/findbugsExcludeFile.xml` (or fix it if you can).

Reply:
Added the field to be excluded on the excludeFile